### ISSUE TITLE
How do Lighthouse scores relate to WCAG 2.1

### ISSUE THREAD

**Issue by Aprahanti1969** [GoogleChrome/lighthouse#9507]
Hi,   

I'm running an accessibility audit of our intranet using the Lighthouse tool in order to comply with legislation which comes into force in the UK in September. This states that all public sector websites (internal and external) must comply with WCAG 2.1 Level AA accessibility guidelines.   

The Lighthouse tool is great and we're getting good scores, but I am not clear on how the scores relate to WCAG 2.1 Level AA.    

Your advice would be much appreciated.   

Thanks,   

Nordin Zaoui

**Comment by patrickhulce** [comment]
Thanks for filing @Aprahanti1969!

tl;dr - Lighthouse covers much, but not all of WCAG 2.1 Level AA requirements and there are things you'll need to check manually.

Lighthouse runs all the wcag2a and wcag2aa tagged rules in axe-core with a few exceptions. 

https://github.com/GoogleChrome/lighthouse/blob/252232db9263750435e471da7ea2da1f97ef0d61/lighthouse-core/gather/gatherers/accessibility.js#L24-L51

[axe-core can't fully automate WCAG 2.1 checks though](https://www.deque.com/blog/support-for-wcag-2-1-in-axe-core/), so there are still going to be some things you need to check. I'd still refer to the [w3 checklists](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa) for final compliance.

**Comment by Aprahanti1969** [comment]
Thanks Patrick,

Regards,

Nordin.
